app/views/demo/actor.html.erb

```diff
@@ -1,52 +1,70 @@
+</ol>
+<% end %>
 
-<p>You can also toggle a feature on or off for <strong>a specific single user</strong> without re-deploying. There are several ways to enable access for users, but for this example, we'll focus on enabling a feature for a specific user rather than a group that includes that user. If you want to enable only select users, you'll generally want to do so by enabling groups that contain the users rather than enabling individual users directly, but we'll get to that shortly.</p>
+<% if !Flipper.enabled?(:demo_actor, current_user) %>
+  <h1>User-based Feature Flipping</h1>
+  <p>You can also toggle a feature on or off for <strong>a specific user</strong>. You can enable access for users several ways, but for this example, we'll focus on enabling a feature for a specific user rather than a group that includes that user.</p>
 
-<p>Previously, we enabled the demo for all users. While that's the simplest case and very handy when a feature is completely ready, you'll occasionally want to enable a feature for a single individual like yourself or a teammate. So let's see how that works.</p>
+  <p>Previously, we enabled the demo for all users. While that's the simplest case and very handy when a feature is completely ready, you'll occasionally want to enable a feature for a single individual like yourself or a teammate. So let's see how that works.</p>
 
-<p><strong><mark>Important:</mark></strong> In order to <a href="https://www.flippercloud.io/docs/features#enablement-actor">enable access for individual users, accounts, or any other "Actor"</a> the model must respond to a <code>flipper_id</code> method that returns a unique identifier. You can implement this method yourself or include the <code>Flipper::Identifier</code> module in your model, and Flipper will use the Class and ID. i.e. <code>User;1</code>.</p>
+  <p>We've named this feature flag <code>demo_actor</code> because it only enables the feature for a single user. In this case, we'll use <code>current_user</code>, but in practice, you could use any instance of a User, Organization, Team, or any "actor" from your system as long as the instance responds to <code>flipper_id</code>.</p>
 
-<fieldset>
-  <legend>Enable Something Just for Yourself</legend>
+  <p><strong><mark>Important:</mark></strong> In order to <a href="https://www.flippercloud.io/docs/features#enablement-actor">enable access for individual users, accounts, or any other actor</a> the model must respond to a <code>flipper_id</code> method that returns a unique identifier. You can implement this method yourself or include the <code>Flipper::Identifier</code> module in your model, and Flipper will use the Class and ID. i.e. <code>User;1</code>. If you want to enable more than a handful of individual users, you'll generally want to do so by enabling groups that contain the users rather than enabling individual users directly, but we'll get to that shortly.</p>
```

Comment:
btw `Flipper::Identifier` gets included into ActiveRecord models by default. I think this can either be cut, or shortened and link to docs somewhere.